templates/ _helpers.tpl

```diff
@@ -73,3 +73,123 @@ Create the name of the service account to use
 {{ "false" }}
 {{- end -}}
 {{- end }}
+
+{{/*
+Given the $.tonicSsoConfig, create all environment variables needed for
+the deployment. This should only be called if $.tonicSsoConfig is populated
+*/}}
+{{- define "tonic.sso" -}}
+{{- $provider := (required "tonicSsoConfig.provider is required for SSO" .provider | lower) -}}
+- name: TONIC_SSO_PROVIDER
+  value: {{ quote .provider }}
+{{- if .groupFilter -}}
+- name: TONIC_SSO_GROUP_FILTER_REGEX
+  value: {{quote .groupFilter}}
+{{- end -}}
+{{- if eq $provider "aws" }}
+{{ include "tonic.sso.aws" . }}
+{{- else if eq $provider "azure" }}
+{{ include "tonic.sso.azure" . }}
+{{- else if eq $provider "duo" }}
+{{ include "tonic.sso.duo" . }}
```

Comment:
I don't see duo on our SSO documentation page, should this be removed?
https://docs.tonic.ai/app/admin/tonic-user-access/single-sign-on